aequilibrae/project/network/network.py

```diff
@@ -346,7 +348,7 @@ def build_graphs(self, fields: list = None, modes: list = None) -> None:
             g.mode = m
             g.network = net
             if centroids.shape[0]:
-                g.prepare_graph(centroids)
+                g.prepare_graph(centroids, lonlat=lonlat)
```

Comment:
We should insert lonlat into the graph regardless of centroids here.  Otherwise it prevents the user from setting a centroid that was not coded as such in the model

Reply:
Resolved in ec0117adfd5584958840d6b2581fccb8249a8d69